documentation/Using-the-SDK/Advanced/Adding-About-Information.md

```diff
@@ -0,0 +1,93 @@
+# Abstract
+
+This document outlines how to add about/author information to your plugins.
+
+# Motivation
+
+When a plugin is used by a large and/or public audience, it is useful to 
+give users an easy way to contact its owners/maintainers. SDK Drivers such as 
+WPA look for __about information__ on a `CustomDataSource` for it to  
```

Comment:
Did you want to update CustomDataSource in this file??

Reply:
I will update it in the other PR assuming this gets merged first.